clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -47,8 +43,7 @@ const (
 
 	currentAPICallMetricsVersion = "v1"
 
-	// TODO(krzysied): figure out why we're getting non-capitalized proxy and fix this
```

Comment:
Is this figured out? :)

Reply:
I recall this was an issue fixed in k/k long time ago. If not, this indicates an issue in k/k which should be fixed. So both ways, we should remove this.